temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingSpanUtils.java

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing.internal;
+
+import com.google.common.base.MoreObjects;
+import io.opentracing.References;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
+
+public class OpenTracingSpanUtils {
+  public static Tracer.SpanBuilder createWorkflowStartSpan(
+      Tracer tracer, long startTimeMs, String spanOperationName, String workflowId) {
+    Map<String, String> tags =
+        new HashMap<String, String>() {
+          {
+            put("workflowTag", workflowId);
+          }
+        };
+    return createSpan(tracer, startTimeMs, spanOperationName, tags, null, References.FOLLOWS_FROM);
+  }
+
+  public static Tracer.SpanBuilder createChildWorkflowStartSpan(
+      Tracer tracer,
+      long startTimeMs,
+      String spanOperationName,
+      String workflowId,
+      String parentWorkflowId,
+      String parentRunId) {
+    Map<String, String> tags =
+        new HashMap<String, String>() {
+          {
+            put("workflowTag", workflowId);
+            put("parentWorkflowId", parentWorkflowId);
+            put("parentRunId", parentRunId);
```

Comment:
These tags should probably be a separate enum or a set of constants so it's easy to see what the full set of tags is.